docs/documentation/usage.rst

```diff
@@ -316,6 +285,108 @@ Several constraint variables can be defined.
 
 Also, see :ref:`get-variable-list`.
 
+***************************
+Advanced configuration file
+***************************
+
+Sys path
+========
+.. code:: yaml
+
+    sys_path:
+      - /path/to/your/module
+
+This section is used to add a path to the Python sys.path.
+This is useful when you have a module that is not in the Python path.
+
+
+Imports
+=======
+.. code:: yaml
+
+    imports:
+        my_driver_1: MyDriver1
+        utils.my_drivers.my_driver_2: MyDriver2
+        utils.my_solvers.my_favourite_solver: MyFavouriteSolver
+
+This section is used to import modules such as solvers and drivers that are not available in OpenMDAO.
+The key is the path of the module or Python file, and the value is the name of the object to import.
+The YAML code lines above will do the equivalent in Python of:
+
+.. code:: python
+
+    from my_driver_1 import MyDriver1
+    from utils.my_drivers.my_driver_2 import MyDriver2
+    from utils.my_solvers.my_favourite_solver import MyFavouriteSolver
+
+
+Driver settings
+===============
+
+The `driver` configuration can be specified in two ways: using the old syntax (a string) or the new syntax (an object).
+
+### Old Syntax
+
+In the old syntax, the driver is specified as a string. This string should be a valid Python expression that creates an instance of an OpenMDAO driver.
+
+```yaml
+driver: om.ScipyOptimizeDriver(tol=1e-2, optimizer='COBYLA')
+```
+
+### New Syntax
+In the new syntax, the driver is specified with an least an instance and more fields such as `options`.
```

Comment:
```suggestion
In the new syntax, the driver is specified with at least an instance and more dict fields such as `options`.
```
(And maybe it's worth insisting even more on the fact that expected attributes are dictionaries)

Reply:
Corrected in [d628427](https://github.com/fast-aircraft-design/FAST-OAD/pull/559/commits/d628427508cd3aaaf7da94b5589df728a6c7705f)